Lib/tkinter/ttk.py

```diff
@@ -570,6 +571,15 @@ def state(self, statespec=None):
         return self.tk.splitlist(str(self.tk.call(self._w, "state", statespec)))
 
 
+    def cget(self, key):
```

Comment:
Please add a test for this :)

Reply:
I've noticed that some PRs have been making a lot of changes to `tkinter` and its tests lately, and I won't add too much to this PR just to prevent some conflicts. I will wait for more people to review the PR before making a decision.